PSReadLine/Changes.txt

```diff
@@ -1,3 +1,26 @@
+### Version 2.1.0-beta1
+
+Pre-release notes:
+
+#### Prototype to support the fish-like auto-suggestion in PSReadLine
+
```

Comment:
You've released it, so it's not a prototype.

How about `Experimental support for fish-like suggestions in PSReadLine`.

Reply:
Updated.